From 8f414526566e207fe255c43cbd644b07fbb51963 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 1 Aug 2018 13:02:38 -0700
Subject: [PATCH] TRITON-635 restify-clients usage should use 'contentMd5'
 option to avoid cross-node version BadDigest errors Reviewed by: Todd
 Whiteman <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 lib/app.js              | 2 +-
 package.json            | 4 ++--
 test/machines/resize.js | 6 ++----
 3 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/lib/app.js b/lib/app.js
index e879b0b..e53f42a 100644
--- a/lib/app.js
+++ b/lib/app.js
@@ -156,7 +156,7 @@ function createHTTPClients(options) {
         imgapi: new SDC.IMGAPI.createClient(options.imgapi),
         keyapi: new keyapi(options),
         cnapi: new SDC.CNAPI(options.cnapi),
-        papi: SDC.PAPI(options.papi),
+        papi: new SDC.PAPI(options.papi),
         mahi: (options.mahi) ? mahi.createClient(options.mahi) : null,
         cns: (options.cns) ? (new SDC.CNS(options.cns)) : null
     };
diff --git a/package.json b/package.json
index c8925f6..b370216 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "cloudapi",
     "description": "SmartDataCenter CloudAPI",
-    "version": "9.2.1",
+    "version": "9.2.2",
     "author": "Joyent (joyent.com)",
     "private": true,
     "engines": {
@@ -35,7 +35,7 @@
         "nodemailer": "0.7.1",
         "nopt": "2.0.0",
         "restify": "4.3.3",
-        "sdc-clients": "11.3.0",
+        "sdc-clients": "11.3.1",
         "semver": "5.4.1",
         "strsplit": "1.0.0",
         "triton-metrics": "0.1.0",
diff --git a/test/machines/resize.js b/test/machines/resize.js
index f6a04ec..435de6a 100644
--- a/test/machines/resize.js
+++ b/test/machines/resize.js
@@ -70,9 +70,8 @@ function (suite, client, other, machine, pkgDown, pkgSame, pkgUp, cb) {
 
 
     suite.test('Resize Machine to same package', function (t) {
-        t.ok(pkgSame, 'Resize same package OK');
+        t.ok(pkgSame, 'Resize same package OK: package ' + pkgSame.name);
 
-        console.log('Resizing to package: %s', pkgSame.name);
         client.post('/my/machines/' + machine, {
             action: 'resize',
             'package': pkgSame.uuid
@@ -88,9 +87,8 @@ function (suite, client, other, machine, pkgDown, pkgSame, pkgUp, cb) {
     });
 
     suite.test('Resize Machine down', function (t) {
-        t.ok(pkgDown, 'Resize down package OK');
+        t.ok(pkgDown, 'Resize down package OK: package ' + pkgDown.name);
 
-        console.log('Resizing to package: %s', pkgDown.name);
         client.post('/my/machines/' + machine, {
             action: 'resize',
             'package': pkgDown.uuid
-- 
2.21.0

